CONTRIBUTING.md

```diff
@@ -99,6 +99,59 @@ This also sets up gulp and Jekyll to watch for file changes to the `/docs`
 and `/src` directories and rebuilds the website accordingly. The command is
 aliased by `npm start`
 
+### Committing files when updating the `/src` directory
+
+When you run `npm start` to preview the website locally, many files are
+generated that are tracked by Git. This leaves your working directory in a dirty
+state and knowing which files need to be included in your commit can seem like a
+daunting task. The following steps can help in committing changes while the team
```

Comment:
I might suggest replacing "The following steps can help in committing changes" with "The following steps will make it easier for you to commit changes while the team looks into different approaches for handling changes to the Standards website and the uswds package."